docs/docs/seo.md

```diff
@@ -64,3 +73,7 @@ After deployment, their [Rich result status reports](https://support.google.com/
 ## Additional resources
 
 You might also be interested in [blog posts about SEO in Gatsby](/blog/tags/seo/).
+
+```
```

Comment:
Seems like that sneaked itself into the PR :)

Reply:
thanks! 👍